client/me/help/help-contact-form/index.jsx

```diff
@@ -206,6 +206,19 @@ export const HelpContactForm = React.createClass( {
 	},
 
 	/**
+	 * Tracks clicks on the questions returned by Sibyl.
+	 * @param {object} event Click event
+	 * @param {object} helpLink Help link, including id for tracking
+	 */
+	trackSibylClick( event, helpLink ) {
+		// bump the clicks stat for the question in MC
+		analytics.mc.bumpStat( 'sibyl_question_clicks', helpLink.id );
+		// track that the user clicked the question, so we can see how
+		// effective questions are at preventing unnessecary chat sessions
+		analytics.tracks.recordEvent( 'calypso_sibyl_question_click' );
```

Comment:
Thank you! I've made the changes and things look like they're working, and I see the action getting the correct help link ID here.

![actions](https://user-images.githubusercontent.com/3314354/29165092-82bb651e-7db9-11e7-8fa7-0da1b96e404c.png)